scripts/tests/run.sh

```diff
@@ -53,6 +53,9 @@ function test_init() {
   echo Superset init
   echo --------------------
   superset init
+  echo Load test users
+  echo --------------------
+  superset load-test-users
```

Comment:
small fix after an earlier refactor where the test script stopped loading test users.